apm-lambda-extension/extension/client.go

```diff
@@ -161,7 +161,7 @@ func (e *Client) InitError(ctx context.Context, errorType string) (*StatusRespon
 	}
 	defer httpRes.Body.Close()
 
-	if httpRes.StatusCode != 200 {
+	if httpRes.StatusCode != 202 {
```

Comment:
Do we care about the precise status code or just whether it was successful?

```suggestion
	if httpRes.StatusCode > 299 {
```

Reply:
Thanks for the comment ! I chose to check against the exact `202` value in the code because that is one of the 4 possible values highlighted by the AWS [docs](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-extensions-api.html#runtimes-extensions-init-error)
<img width="658" alt="image" src="https://user-images.githubusercontent.com/48380853/157666892-3db742c1-b84c-4c0b-bc1d-3cc3c2e45928.png">